### ISSUE TITLE
Weird issue with spaces

### ISSUE THREAD

**Issue by ftyers** [apertium/apertium-separable#19]
```
fran@ipek:~/source/apertium/pairs/apertium-quc-spa$ echo "Jas che mna kixpe chwe’q?" | apertium -d . quc-spa
por qué  *mna vinisteis a @we’*q?
fran@ipek:~/source/apertium/pairs/apertium-quc-spa$ echo "Jas  che mna kixpe chwe’q?" | apertium -d . quc-spa
  @jasche  *mna vinisteis a @we’*q?

```

**Comment by mr-martian** [comment]
This appears to have been fixed by #25.

**Comment by mr-martian** [comment]
@ftyers could you check whether this is still an issue?
